packages/usa-validation/src/index.js

```diff
@@ -1,14 +1,47 @@
 const behavior = require("../../uswds-core/src/js/utils/behavior");
 const validate = require("../../uswds-core/src/js/utils/validate-input");
+const { prefix: PREFIX } = require("../../uswds-core/src/js/config");
+const selectOrMatches = require("../../uswds-core/src/js/utils/select-or-matches");
+
+const VALIDATE_INPUT = "input[data-validation-element]";
+const CHECKLIST_ITEM = `.${PREFIX}-checklist__item`;
 
 function change() {
   validate(this);
 }
 
-const validator = behavior({
-  "keyup change": {
-    "input[data-validation-element]": change,
+function createHiddenLabel() {
+  const hiddenLabel = document.createElement("span");
+
+  hiddenLabel.classList.add("usa-sr-only");
+  hiddenLabel.textContent = "Incomplete";
+  hiddenLabel.setAttribute("data-checklist-label", "");
+
+  return hiddenLabel;
+}
+
+const validator = behavior(
+  {
+    "input change": {
+      [VALIDATE_INPUT]: change,
+    },
   },
-});
+  {
+    init(root) {
+      selectOrMatches(VALIDATE_INPUT, root).forEach((item) => {
+        const validationParent = item.parentNode;
```

Comment:
Since the input now controls the checklist, it might make sense to try adding `aria-controls=""` and point to the checklist `ID`.

Reply:
Resolved in [23a5c3f](https://github.com/uswds/uswds/pull/4719/commits/23a5c3f55d886c5437f123235922d7f1cb4c98d7)